### ISSUE TITLE
Remove redundant * when use count()

### ISSUE THREAD

**Issue by jk2K** [FaaPz/PDO#10]
``` php
$selectStatement = new SelectStatement($this->pdo, array('*'));
// or
$selectStatement = $this->pdo->select();

$sel = $selectStatement->from('dede_archives')->count();
echo $sel;
```

result

``` sql
SELECT * , COUNT( * ) FROM dede_archives
```

maybe in front of \* can be removed

**Comment by FaaPz** [comment]
How is this code related to Slim-PDO, @jk2K? It just looks like you're using the Yii 2 built-in database class...

I briefly tested your code myself, and it's working correctly only using Yii 2.

**Comment by jk2K** [comment]
sorry, I have updated the issue

**Comment by FaaPz** [comment]
It's fixed! :smile: (100fec2be669a89d42cb33e8decb1775584a2507)

**Comment by jk2K** [comment]
awesome, thanks 🙏
